clang/test/Driver/amdgpu-openmp-toolchain.c

```diff
@@ -11,22 +11,10 @@
 
 // verify the tools invocations
 // FIXME: Need to resolve the conflict if the test is turned on
-<<<<<<< HEAD
```

Comment:
I think this is not intended here

Reply:
the changes in this file are from sycl branch. They will be fixed once sycl branch is fixed.

Reply:
I agree. Making changes like this that are simply unrelated to the other changes might just serve to delay the patch more. Do we need it for this patch or can we do it separately?

Reply:
synced the file with sycl branch.